data_diff/sql.py

```diff
@@ -113,8 +113,13 @@ class Checksum(Sql):
     exprs: List[SqlOrStr]
 
     def compile(self, c: Compiler):
-        compiled_exprs = ", ".join(c.database.to_string(s) for s in map(c.compile, self.exprs))
-        expr = f"concat({compiled_exprs})"
+        compiled_exprs_raw = [c.database.to_string(s) for s in map(c.compile, self.exprs)]
```

Comment:
Could you provide an example here or in the description of what prompted this change?